modules/nf-core/picard/collecthsmetrics/main.nf

```diff
@@ -31,13 +33,47 @@ process PICARD_COLLECTHSMETRICS {
         avail_mem = (task.memory.mega*0.8).intValue()
     }
     """
+
+    if [[ "$bait_intervals" == *.bed || "$bait_intervals" == *.bed.gz ]]; then
+    bait_bed=$bait_intervals
+    bait_intervals=\${bait_bed%.bed*}.interval_list
```

Comment:
can't we deal with that in the script with the usual `def`?

Reply:
Ah, never thought of that 🤦🏻

Reply:
Should be good to go now!